lineman/app/components/group_page/cover_photo_form/cover_photo_form.haml

```diff
@@ -0,0 +1,21 @@
+.group-cover-photo-modal
+  .lmo-disabled-form{ng-show: 'isDisabled'}
+  .modal-header
+    %button{type: 'button', class: 'close', ng-click: '$close()'}
+      %span{aria-hidden: true} &times;
+      %span.sr-only{translate: 'common.action.cancel'}
+    %h1.lmo-h1.modal-title{translate: 'group_cover_modal.heading'}
+
+  .modal-body
+    %a.lmo-link{href: '', ng-click: 'selectFile()'}
```

Comment:
another a11y think. Could you make this a button element with the .lmo-btn-link class